consensus/config/src/committee.rs

```diff
@@ -45,10 +45,10 @@ impl Committee {
             authorities.len()
         );
 
-        let total_stake = authorities.iter().map(|a| a.stake).sum();
+        let total_stake = authorities.iter().map(|a| a.stake).sum::<u64>();
         assert_ne!(total_stake, 0, "Total stake cannot be zero!");
         let quorum_threshold = 2 * total_stake / 3 + 1;
-        let validity_threshold = (total_stake + 2) / 3;
+        let validity_threshold = total_stake.div_ceil(3);
```

Comment:
Not the same calculation?

Reply:
It is, see https://rust-lang.github.io/rust-clippy/master/index.html#/div_ceil

Reply:
Oh, okay then